tests/serverspec/default.yml

```diff
@@ -67,8 +67,8 @@
   roles:
     - ansible-role-virtual-interface
   vars:
-    virtual_interface:
-      gre0:
+    virtual_interfaces:
```

Comment:
in general, a list variable should be plural. this change, however, is both backward-incompatible and contrary to general expectations, assuming variables in a role always start with role name.